src/app/pcasts/models/ignored_users.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import UniqueConstraint
+from . import *
+
+class IgnoredUsers(Base):
+  __tablename__ = 'ignored_users'
+  __table_args__ = (
+      UniqueConstraint('user_id'),
+  )
+
+  user_id = db.Column(db.Integer, \
+      db.ForeignKey('users.id', ondelete='CASCADE'), primary_key=True)
+  ignored_fb_ids = db.Column(db.Text)
```

Comment:
I'll run some benchmarks/ask someone more knowledgeable about this.
I would imagine there aren't that many people you want to ignore so a pure relational way of storing
user_id, ignored_id might be more optimal than the string.

Reply:
If we're adding a string for each user, why wouldn't we add the `ignored_fb_ids` to the user model instead?